addons/mail/static/src/models/volume_setting/volume_setting.js

```diff
@@ -9,37 +9,6 @@ function factory(dependencies) {
 
     class VolumeSetting extends dependencies['mail.model'] {
 
-
-        //----------------------------------------------------------------------
-        // Public
-        //----------------------------------------------------------------------
-
-        /**
-         * @static
-         * @param {Object} data
-         * @returns {Object}
-         */
-        static convertData(data) {
```

Comment:
@seb-odoo what about this removal? Is it fine to change the server response format in 15.0?